packages/css/src/components/selectable/selectable.scss

```diff
@@ -5,3 +5,7 @@
     @include selectable_notLastChild();
   }
 }
+
+.dso-selectable-nested-parent {
```

Comment:
Wat is de relatie tussen `-parent` en `-children`, en waarom is er een mixin `-root()` voor nodig?

Reply:
Ik was schijnbaar in de veronderstelling dat er een fysieke lijst gemaakt moest worden, omdat het anders niet netjes zou tonen. Na opschoning van deze `ul li` constructie is dit stukje stijling inclusief deze mixin niet meer nodig.